Cargo.toml

```diff
@@ -17,6 +17,8 @@ edition = "2018"
 [dependencies]
 cfg-if = "1.0"
 rayon = { version = "^1.5.1", optional = true }
+serde = { version = "1.0.152", features = ["derive"], optional = true }
+serde_with = { version = "2.1.0", optional = true }
```

Comment:
What is it used for?

Reply:
I was gonna try to use a feature in the serde_with crate but it wouldn't have done what I thought. It'll get removed. And of course serde with serde with derive feature enabled. Both optionally compiled of and only if you enable the serde feature when compiling sysinfo.